packages/teams-js/src/public/dialog.ts

```diff
@@ -95,7 +95,7 @@ export namespace dialog {
   }
 
   /**
-   * Submit the dialog module.
+   * Submit the dialog module and closes the dialog
```

Comment:
This should be "and close the dialog" to match tenses

Reply:
you're right!